src/view/pane/chat.js

```diff
@@ -550,10 +550,11 @@ Candy.View.Pane = (function(self, $) {
        * Show the login form modal
        *
        * Parameters:
-       *  (String) message - optional message to display above the form
+       *  (String) messageKey - optional message to display above the form
        *  (String) presetJid - optional user jid. if set, the user will only be prompted for password.
        */
-      showLoginForm: function(message, presetJid) {
+      showLoginForm: function(messageKey, presetJid) {
+        var message = $.i18n._(messageKey);
```

Comment:
here's the place to do: 

```javascript
var message = messageKey ? $.i18n._(messageKey) : '';
```
Then you can remove the ternary operator for `message` on the line with `self.Chat.Modal.show` and you won't need the default assignments from above.